docroot/themes/custom/uids_base/scss/components/typography/headings.scss

```diff
@@ -61,3 +61,9 @@ body:not(.page-node-type-article, .page-node-type-page) #block-uids-base-page-ti
     background: $primary;
   }
 }
+
+// @todo remove in UIDS.
```

Comment:
```suggestion
// @todo remove in UIDS.
// https://github.com/uiowa/uids/issues/876
```

Reply:
Not sure I understand this. I think this was related to the removal of the `bold-headline` class. I believe that has already happened in UIDS 4. Would need to verify.

Reply:
To be clear, I think we can remove all the `bold-headline` references we find, since we've had a replacement for that for a long time.

Reply:
I just removed it and adjusted the scope of it to `.card .headline`.   We are declaring a line-height property in UIDS that is interfering with the preferred line-height of the card headline.

Reply:
Is there follow-up? Can we remove the todo?

Reply:
Todo has been removed.  No follow-up for now.